docs/en/manuals/html5.md

```diff
@@ -71,213 +64,157 @@ Internet Explorer 11 (Full screen)
-
-Unless overridden, Defold will produce a default page, consisting of:
+## Customizing HTML5 bundle
 
-- An HTML5 canvas for your application
-- A button to access fullscreen mode
-- Splash screen content and logic
-- Development tools, if required
+When generating an HTML5 version of your game, Defold provides a default web page. It references style and script resources that dictate how your game is presented.
 
 Each time the application is exported, this content is created afresh. If you wish to customize any of these elements you must make modifications to your project settings. To do so, open the *game.project* in the Defold editor and scroll to the *html5* section:
 
-![HTML5 settings](images/html5/html5_styling_settings.png)
+![HTML5 Section](images/html5/html5_section.png)
 
-These settings allow you to add a custom HTML file, stylesheet and splash image.
+More information about every option is available in [project settings manual](/manuals/project-settings/#_html5).
 
-::: sidenote
-To get started, you can begin by exporting the application once with the default settings. You then copy the exported HTML and CSS back into your project. Once copied, make the changes to the settings, as described above, and start editing the files.
+::: important
+You can't modify files of the default html/css template in `builtins` folder. For applying your modifications copy/paste needed file from `builtins` and set this file in `game.project`.
 :::
 
-You have considerable freedom to make changes or additions to this content, but some restrictions apply:
-
-- The canvas should not be styled with any border or padding. If you do, mouse input coordinates will be wrong.
-- If Facebook support is required then the script tag that loads the Javascript SDK should appear before the main application script.
-
-## Splash screens
-
-By default, a splash screen is implemented. If you wish to customize or remove the splash screen then there are a few features that require attention:
-
-- HTML markup
-- CSS styling
-- Javascript logic
-
-Although the implementation of any associated Javascript is a matter of preference, Defold applications will send information back to you if you implement a "SplashControl" object that contains the following methods:
+::: sidenote
+The canvas shouldn't be styled with any border or padding. If you do, mouse input coordinates will be wrong.
+:::
 
-onSetMessage(text)
-: This callback receives text that is intended for display to the user.
+In `game.project` possible to turn-off the `Fullscreen` button and `Made with Defold` link.
+Defold provides a dark and light theme for the index.html. The light theme is set by default but it possible to change just changing `Custom CSS` file. Also, was predefined four different scale modes that possible to change in `Scale Mode` field.
```

Comment:
"but it possible to change just changing" -> "but it is possible to change by changing"

"Also, was predefined four different scale modes that possible to change in `Scale Mode` field.
" -> "There is also four predefined scale modes to chose from in the `Scale Mode` field."